src/main.rs

```diff
@@ -180,11 +194,23 @@ pub async fn main() {
                 let max_rows = header.extrinsics_root.rows;
                 let max_cols = header.extrinsics_root.cols;
                 let commitment = header.extrinsics_root.commitment.clone();
-
+				
+				if max_cols<3{
+					log::info!("lower number of chunks");
+					std::process::exit(0);
```

Comment:
@MiguelDD1 Should we avoid such exits?

Reply:
My suggestion is to use [thiserror create](https://crates.io/crates/thiserror) and/or [Anyhow](https://github.com/dtolnay/anyhow) for error handling.